solutions/community-deploy/cloud/AWS/variable.tf

```diff
@@ -41,7 +41,6 @@ variable "key_name" {
 variable "instance_count" {
   description = "EC2 instance count"
   type        = number
-  default     = "1"
```

Comment:
Please keep the default as 1 so that if no values are provided for `instance_count` then it will be used.

Reply:
fixed